public/apps/account/utils.tsx

```diff
@@ -31,7 +31,7 @@ export async function fetchAccountInfoSafe(http: HttpStart): Promise<AccountInfo
 export async function logout(http: HttpStart, logoutUrl?: string): Promise<void> {
   await httpPost(http, API_AUTH_LOGOUT);
   setShouldShowTenantPopup(null);
-  window.location.href = logoutUrl || http.basePath.serverBasePath;
+  window.location.href = logoutUrl || `${http.basePath.serverBasePath}/app/login`;
```

Comment:
`/app/login` is only accessible for basic auth, for other auth types like SAML and OIDC, the login is handled by IDP login page, not Kibana. Accessing the root path will trigger the login workflow for all auth types, so I would recommend not to default redirect to basic auth login page

Reply:
The SAML and OIDC flow are not disturbed. Reference : https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/main/public/apps/account/log-out-button.tsx#L27-L40, seems like the logout method is only called when we are using ***basicauth***